Native_UWP_V2/MainPage.xaml.cs

```diff
@@ -170,6 +170,7 @@ private void DisplayBasicTokenInfo(AuthenticationResult authResult)
             {
                 TokenInfoText.Text += $"User Name: {authResult.Account.Username}" + Environment.NewLine;
                 TokenInfoText.Text += $"Token Expires: {authResult.ExpiresOn.ToLocalTime()}" + Environment.NewLine;
+                TokenText.Text += $"{authResult.AccessToken}";
```

Comment:
No, we cannot display the token @bgavrilMS : we are not allowed to do that.
Please revert this PR.

Reply:
Ok. I guess the B2C samples does that though.